src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommandCommandInfo.cs

```diff
@@ -92,7 +92,7 @@ public ShowCommandCommandInfo(PSObject other)
                 var parameterSets = (other.Members["ParameterSets"].Value as PSObject).BaseObject as System.Collections.ArrayList;
                 this.ParameterSets = GetObjectEnumerable(parameterSets).Cast<PSObject>().Select(x => new ShowCommandParameterSetInfo(x)).ToList().AsReadOnly();
 
-                if (other.Members["Module"] != null && other.Members["Module"].Value as PSObject != null)
+                if (other.Members["Module"] != null && other.Members["Module"].Value is PSObject)
```

Comment:
```suggestion
                if (other.Members["Module"]?.Value is PSObject)
```

We can simplify this, I think?